config/config.go

```diff
@@ -22,23 +22,29 @@ type Config struct {
 	Orchestrator     string            `short:"o" long:"orchestrator" description:"Container orchestrator to use." env:"BIVAC_ORCHESTRATOR"`
 	TargetURL        string            `short:"u" long:"target-url" description:"The target URL to push to." env:"BIVAC_TARGET_URL"`
 	CheckEvery       string            `long:"check-every" description:"Time between backup checks." env:"BIVAC_CHECK_EVERY" default:"24h"`
-	RemoveOlderThan  string            `long:"remove-older-than" description:"Remove backups older than the specified interval." env:"BIVAC_REMOVE_OLDER_THAN" default:"30D"`
 	LabelPrefix      string            `long:"label-prefix" description:"The volume prefix label." env:"BIVAC_LABEL_PREFIX"`
 	ExtraEnv         map[string]string `long:"extra-env" description:"Extra environment variables to share with workers." env:"BIVAC_EXTRA_ENV"`
 	ProvidersFile    string            `short:"p" long:"providers-file" description:"Path to providers configuration file." env:"BIVAC_PROVIDERS_FILE" default:"/providers-config.default.toml"`
 
 	Restic struct {
-		Image    string `long:"restic-image" description:"The restic docker image." env:"RESTIC_DOCKER_IMAGE" default:"restic/restic:latest"`
-		Password string `long:"restic-password" description:"The restic backup password." env:"RESTIC_PASSWORD"`
+		CommonArgs []string `long:"restic-args" description:"Arguments to pass to restic engine." env:"RESTIC_COMMON_ARGS" default:"-r %B/%P/%V"`
```

Comment:
The type of `CommonArgs` (like all the *Args variables) cannot be `[]string`. If you use the environment variable `RESTIC_COMMON_ARGS="foo bar"`, the value of `CommonArgs` will be `[]string{"foo bar"}` and I guess it's not what you expect.
I think the best solution here is to use the type `string` and split the string before using it.

Reply:
Unless you change the env delimiter, but that behavior might even be worse...

Reply:
I've thought about that and it would work very well with environment variables. The problem is that if you're using the CLI, you will have to write something like `--restic-args=init --restic-args=-r --restic-args=s3:my-repo` and it would be a nightmare.